lib/gitsh/cli.rb

```diff
@@ -82,5 +84,15 @@ def option_parser
         end
       end
     end
+
+    def ensure_executable_git
+      IO.popen(env.git_command).close
+    rescue Errno::ENOENT
+      env.puts_error(
+        "gitsh: No such executable git: #{env.git_command}. Please " \
+        'specify an executable git using the --git option.',
+      )
+      exit EX_DATAERR
```

Comment:
Maybe `EX_UNAVAILABLE` (69) would be more appropriate here?

> A service is unavailable.  This can occur if a support program or file does not exist.